datahub-web-react/src/app/shared/admin/HeaderLinks.tsx

```diff
@@ -73,11 +73,14 @@ export function HeaderLinks(props: Props) {
         isIngestionEnabled && me && me.platformPrivileges.manageIngestion && me.platformPrivileges.manageSecrets;
     const showDomains = me?.platformPrivileges.createDomains || me?.platformPrivileges.manageDomains;
     const showASD = true;
+    const root = '/';
+    const cipEnv = window?.location?.hostname?.split('.')[0] || root;
+    const ASDUrl = cipEnv.startsWith('cip-') ? `https://search-${cipEnv}.tax.service.gov.uk` : root;
```

Comment:
this should be search-cip-<env>

Reply:
`cipEnv` is made up of two parts, `cip-` suffix with the environment . g.e. `cip-qa`.. I've added a comment to describe the variable